AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -158,7 +158,7 @@ final class InitialSyncOperation: AsynchronousOperation {
         recordsReceived += UInt(items.count)
 
         for item in items {
-            reconciliationQueue.offer(item)
+            reconciliationQueue.offer(item, isFullSync: lastSyncTime == nil)
```

Comment:
I should decide the syncType based on `lastSyncTime` which can only be checked in `initialSyncOperation` class because DataStoreConfiguration is only accessible in this class: https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift#L78, and to let `AWSModelReconciliationQueue` class to know the `syncType` and `count`, I need a way to pass these two information. 

If the sync type for every item need to be reconciled are the same (Maybe it's not, correct me if I am wrong), and I don't find another way to track the sync type of the entire operation.